### PR TITLE
docs: Add docker compose workflow for running docs locally

### LINKED ISSUE #5399: document how to perform local development on docs-v2 using docker compose
Document the use of docker or docker compose to do local development on docs-v2.

##### Relevant URLs

* https://docker.hugomods.com/docs/development/docker-compose/

### PR DESCRIPTION
**Pull request by waynr**
Closes #5399

Definitely open to feedback here. It seems to just work when I run `docker compose up local-dev` -- the rendered site looks fine without running any additional npm commands or whatever.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,3 +52,18 @@ including our GPG key, can be found at https://www.influxdata.com/how-to-report-
...
+
+2. **Install Docker & Compose Plugin**
+
+3. **Start the Hugo Server**
+
+  In your terminal use `docker compose` to start the Hugo server in dev mode:
+
+    ```sh
+    docker compose up local-dev
+    ```
```
Comment (jstirnaman) [inline README.md:68]:
```suggestion
3. Use Docker Compose to start the Hugo server in development mode--for example, enter the following command in your terminal:

    ```sh
    docker compose up local-dev
    ```
```

File: README.md [hunk 2]
```diff
@@ -52,3 +52,18 @@ including our GPG key, can be found at https://www.influxdata.com/how-to-report-
       npx hugo server
       ```
 5. View the docs at [localhost:1313](http://localhost:1313).
+
+### Alternative: Use docker compose
+
+1. [**Clone this repository**](https://help.github.com/articles/cloning-a-repository/) to your local machine.
```
Comment (jstirnaman) [inline README.md:58]:
```suggestion
1. Clone this repository to your local machine. See how to [clone a repository](https://help.github.com/articles/cloning-a-repository/).
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Thanks for adding this! I've been meaning to get to it for a while.

I just made some style suggestions for the README and some minor changes to make the compose file more explanatory.

**Comment by waynr** [comment]
> I just made some style suggestions for the README and some minor changes to make the compose file more explanatory.

For what it's worth I was following the existing convention from the previous in the README but I like your suggestions better so I accepted them.

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**